.github/workflows/docker-push-ghcr.yaml

```diff
@@ -2,6 +2,11 @@
 name: Push docker image to ghcr
 on:
   workflow_call:
+    inputs:
+      build_with_profiler:
+        type: boolean
+        required: false
+        default: false
```

Comment:
Why can't we add an optional parameter for build-args and just pass it to the Docker job? This approach will be more transparent and have more chances to be re-used in the future.

Reply:
This optional parameter will always be tied to the "Build and push with profiler enabled" build step. I can't see how it could be reused.

Reply:
I don't quite follow this. Could you please say more?

I see that these parameters could be used something like this:
```yaml
  push:

    name: "Build an image with pprof"
    uses: networkservicemesh/.github/.github/workflows/docker-push-ghcr.yaml@main
    with:
       docker-target: pprof
    name: "Build a production image"
    uses: networkservicemesh/.github/.github/workflows/docker-push-ghcr.yaml@main
    with:
       docker-target: build
```

Reply:
Fixed, please re-review